tap_pendo/streams.py

```diff
@@ -21,147 +21,18 @@
 KEY_PROPERTIES = ['id']
 BASE_URL = "https://app.pendo.io"
 
-endpoints = {
-    "account": {
-        "method": "GET",
-        "endpoint": "/api/v1/account/{accountId}"
-    },
-    "accounts": {
-        "method": "POST",
-        "endpoint": "/api/v1/aggregation",
-        "data": {
-            "response": {
-                "mimeType": "application/json"
-            },
-            "request": {
-                "name": "all-accounts",
-                "pipeline": [{
-                    "source": {
-                        "accounts": "null"
-                    }
-                }],
-                "requestId": "all-accounts"
-            }
-        }
-    },
-    "features": {
-        "method": "POST",
-        "endpoint": "/api/v1/aggregation",
-    },
-    "guide_events": {
-        "method": "POST",
-        "endpoint": "/api/v1/aggregation",
-    },
-    "feature_events": {
-        "method": "POST",
-        "endpoint": "/api/v1/aggregation",
-        "data": {
```

Comment:
Where did you mention this data? As I can't see the data field in the Endpoints class.

Reply:
The 'data' key was not being used anywhere in the code, there was a get_body function that was being used. So haven't included that attribute in the class